### PR TITLE
generate sha for all istioctl archive

### PR DESCRIPTION
**Pull request by irisdingbj**
https://github.com/istio/istio/issues/15846

[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[X] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: release/gcb/gcb_lib.sh [hunk 1]
```diff
@@ -96,15 +96,12 @@ function make_istio() {
...
-  
-  mapfile -t ISTIOCTL_ARCHIVES < <(ls "${ISTIO_OUT}"/archive/istioctl*.tar.gz)
-  mapfile -t ISTIOCTL_ARCHIVES < <(ls "${ISTIO_OUT}"/archive/istioctl*.zip)
-
-  for i in "${ISTIOCTL_ARCHIVES[@]}"; do
-    sha256sum "$i" > "$i.sha256"
+  for file in "${ISTIO_OUT}"/archive/istioctl*.tar.gz; do
+    sha256sum "${file}" > "$file.sha256"
+  done
+  for file in "${ISTIO_OUT}"/archive/istioctl*.zip; do
```
Comment (howardjohn) [inline release/gcb/gcb_lib.sh]: why not just `istioctl*` and one loop? lgtm though

Comment (irisdingbj) [reply release/gcb/gcb_lib.sh]: changed to one loop

**Pull request merged**